pkg/virt-api/webhooks/validating-webhook/admitters/preference-admitter.go

```diff
@@ -74,6 +92,26 @@ func validateSpreadOptions(field *k8sfield.Path, spec *instancetypeapiv1beta1.Vi
 	return nil
 }
 
+const deprecatedPreferredCPUTopologyErrFmt = "PreferredCPUTopology %s is deprecated for removal in a future release, please use %s instead"
+
+func checkForDeprecatedPreferredCPUTopology(spec *instancetypeapiv1beta1.VirtualMachinePreferenceSpec) []string {
+	if spec.CPU == nil || spec.CPU.PreferredCPUTopology == nil {
+		return nil
+	}
+	deprecatedTopologies := map[instancetypeapiv1beta1.PreferredCPUTopology]instancetypeapiv1beta1.PreferredCPUTopology{
```

Comment:
Might make sense to make it an unexported var in the pkg? So it is initialized only once.